section2/reflection.md

```diff
@@ -2,28 +2,87 @@
 
 1. Regarding the blog posts in Part A, how do you feel about asking questions? Do you tend to ask them too soon, or wait too long, or somewhere in between?
 
+I'm still getting used to Googling first & then asking questions on the Slack channel if I'm still stuck. My first instinct is to ask someone else a question, so I'm working on always searching Google first before asking others. I would say I ask them too soon normally but now am somewhere in between.
```

Comment:
Don't fight this instinct. It is okay to want to ask others. As you begin writing that Slack message asking for help, try to follow this format
1. what is the issue
2. what did you try/resources **(here is where you can pause in your question asking and attempt a quick google search. This is also a good habit because even if YOU solve this issue, you now have documentation you can share with the group about this error!)**
3. code snippets/screenshots